roles/deploy_bmh/tasks/main.yml

```diff
@@ -44,6 +44,19 @@
       vars:
         node_name: "{{ item.key }}"
         node_data: "{{ item.value }}"
+        _credentials_content: >-
+          {{
+            lookup('file', item.value['credentials_file']) | from_yaml
```

Comment:
beware of the `lookup()` - it's running from the ansible controller. I think this is actually what you want, but I'd rather double-check that fact.

Reply:
Yes, that's the case. I'll copy that one from the jumpbox into the controller-0.